core/tellie_server.py

```diff
@@ -852,9 +852,9 @@ def command_pulse_width(par):
 
 def command_pulse_number(par):
     """Get the command to set a pulse number"""
+    par = int(par)
     if par > p._max_pulse_number or par < 0:
         raise TellieException("Invalid pulse number: %s" % (par))
-    par = int(par)
     #parameters  = ParametersClass()
     adjusted, actual_par, hi, lo = p.pulse_number(par)
     if adjusted is True:
```

Comment:
Why would we want to raise an exception if the pulse number has been adjusted? This is really odd because the pulse number cannot be prime if it is above 255. It has to do with `hi` and `lo` being two separate parameters. Do we want to worry about this?